Content.Server/Cognislime/CognislimeSystem.cs

```diff
@@ -0,0 +1,108 @@
+using Content.Shared.DoAfter;
+using Content.Shared.Examine;
+using Content.Shared.Interaction;
+using Content.Shared.Popups;
+using Content.Shared.Emoting;
+using Content.Shared.Mind.Components;
+using Content.Shared.Movement.Components;
+using Content.Shared.Speech;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Cognislime;
+using Content.Shared.Whitelist;
+using Content.Shared.Interaction.Components;
+using Content.Server.Ghost.Roles.Components;
+using Content.Server.Resist;
+
+namespace Content.Server.Cognislime;
+
+/// <summary>
+/// Makes objects sentient.
+/// </summary>
+public sealed class CognislimeSystem : SharedCognislimeSystem
```

Comment:
the doafter should be moved to shared for prediction only the ghost role stuff has to stay here